### DISCUSSION TITLE
When I run locally the command with newman the collection is succesfull I get the report, everything, but not in workflow.

### DISCUSSION THREAD

**Discussion by george-fron-db** [community/community#50323] [Actions] [answered]
### Select Topic Area

Question

### Body

Ok so I am at my wits end. This is a bit frustrating as it doesn't tell me anything, this error is just...horrible. Locally the command works flawlessly, but on github it just gives me this :
"Error: Process completed with exit code 1."

      - name: Checkout private repo
        uses: actions/checkout@v3
            
      - name: SetUp Node
        uses: actions/setup-node@v3
        with:
          node-version: 16
          
      - name: Install newman
        run: |
              npm install -g newman
              npm install -g newman-reporter-htmlextra
              npm fund

      - name: Run Collection
        run: |
              newman run "ApigeeServicesRequests.postman_collection.json" -e "${{ inputs.Environment }}.postman_environment.json" --env-var "access_token=${{inputs.Token}}" --folder "${{ inputs.Package }}" -r htmlextra --reporter-htmlextra-export Report/report.html -k

      - name: Publish Report
        uses: actions/upload-artifact@v3
        with:
          name: ${{inputs.Environment}}_${{inputs.Package}}
          path: Report

This is the yml file. I can't share more due to company policies, all I can say is that I am inside a maven container and the workflow stops at 
<img width="1368" alt="Screenshot 2023-03-17 at 15 40 11" src="https://user-images.githubusercontent.com/102583616/225921337-04c2eee4-e64e-42d1-a693-49efebc0bb3f.png">
Any ideas? Also, if I run a "wrong" folder, like something that doesn't exist, it will run the workflow without error.

**Comment by airtower-luna** [comment]
Any exit code other than zero indicates the program you called had some kind of error. Obviously without seeing the actual error messages from the program itself it's impossible to say why. All I can say is: Double check that the command, input files, etc. are correct. It might make sense to run the commands locally and see what happens.

**Comment by Mohamed-SayedAlAhl** [comment]
The "Error: Process completed with exit code 1" message usually indicates that the program called by the workflow step failed for some reason. Without seeing the actual error message from newman, it is difficult to identify the specific issue. However, here are a few things you could try:

- Double check that the command and input files are correct, as suggested by airtower-luna. Make sure that the collection and environment files are in the correct directory and that their names match what is specified in the workflow.
- Check if the version of newman on GitHub matches the version you have installed locally. You can specify the version of newman to install using the "--global newman@<version>" flag in the "npm install" command.
- Add the "-d" or "--disable-unicode" flag to the newman command in the workflow, as some users have reported issues with Unicode characters causing newman to fail.
- Try running the workflow on a different runner or operating system to see if the issue is specific to the current environment.

If none of these solutions work, you may need to provide more details about the error message or seek further assistance from the newman community or GitHub support.

**Comment by george-fron-db** [comment, marked as answer]
First of all thank you all for replying and trying to help. I managed to pass this error by adding "--insecure" to the command line.
I hope this helps others in the future.
